docs/source/best-practices/scratchpurge.md

```diff
@@ -1,8 +1,76 @@
 (scratch-directory-purge)=
-# Preparing for a /scratch purge
+# Scratch Directory Purge
 
-The space assigned to every user in /scratch/<username> is not meant for persistant storage and is purged every month by the Research Computing team. Files are not backed up in /scratch and as such important data or scripts need to be transfered quickly. 
+The space assigned to every user in `/scratch/<username>` is not meant for persistant storage and is purged every month by the Research Computing team. Files are not backed up in /scratch and as such important data or scripts need to be transfered quickly to /work or /home to be retained.
 
-The /scratch space is intended to provide temporary storage for jobs that produce a lot of output, not all of which will be retained. 
+:::{important}
+The /scratch space is intended to provide temporary storage for jobs that produce a lot of output, not all of which will be retained.
+:::
 
-We recommend 
\ No newline at end of file
+There are several things you can do to prepare for a purge of /scratch.
+
+1. Transfer any materials that you need to save to /work or /home.
+
+Example using `srun`
+
+:::{code-block}
+srun --pty /bin/bash
+mv /scratch/<username>/file_to_keep.out /home/<username>/
+:::
+
+Example moving files via an `sbatch` script.
+
+You will need to copy the script below to a text file (e.g., filemover.sh) and submit it to the scheduler with the command `sbatch filemover.sh`
+
+:::{code-block}
+#!/bin/bash
+
+#SBATCH --nodes=1
+#SBATCH --time=05:00:00
+#SBATCH --partition=short
+#SBATCH --job-name=moveit
+#SBATCH --ntasks=1
+
+# if statement to check if tar.gz file exists, delete the output path
+
+mv /scratch/file_to_keep /work/<groupname>/myimportantdata
+
+:::
+
+2. On the day of the /scratch purge you will not be able to write job outputs to /scratch. Please edit your sbatch scripts to write outputs to /work.
+
+3. If you have jobs that continually write output to scratch and run for long periods of time, please make sure you are [checkpointing](https://rc-docs.northeastern.edu/en/latest/best-practices/checkpointing.html). This will allow the resumption of your jobs around the /scratch purge.
+
+4. If you wish to retain entire directories that were generated in /scratch as part of a job output, you can tar the directory first and move the compressed file to your /home or /work. We recommned only doing this if the directory will not be opened often as for large or many files taring and untaring can be time consuming.
+
+:::{code-block}
+# First get on a compute node
+srun --pty /bin/bash
+
+tar czvf name_of_output.tar.gz /scratch/<username>/directory
```

Comment:
tar -czvf

Reply:
the dash isn't required, but I put it in in case it feels better to others as well.